coopr-ui/app/features/clusters/controllers/detail-ctrl.js

```diff
@@ -137,6 +153,13 @@ function ($scope, MYSERVICEPICKER_EVENT, CrudFormBase, $state, $modal, $alert, m
 
   /* ----------------------------------------------------------------------- */
 
+  function showLeaseAlert() {
+    $alert({
+      title: 'Lease extension maxed: ',
+      content: 'maximum lease is ' + $scope.leaseMaxHumanized,
```

Comment:
thought leaseMaxHumanized never changes so we should construct it once, added a variable instead of declaring on scope. Moment's humanize() produces something like "maximum lease is 3 months" which is not specific enough.